src/web/src/components/SearchProvider.tsx

```diff
@@ -35,6 +35,19 @@ const SearchContext = createContext<SearchContextInterface>({
   },
 });
 
+// Returns the string in the filter accordingly after loading results.
+function getFilter(filter: any) {
+  switch (filter) {
+    case 'author':
+      return 'author';
+    case 'date':
+      return 'date';
+    case 'post':
```

Comment:
Since we default the search option to `'post'`, this is probably not needed and can be removed.

Reply:
Let me ping you about this, because I had some difficulty understanding the login on line 50 of the original.